mmv1/products/notebooks/terraform.yaml

```diff
@@ -116,6 +116,67 @@ overrides: !ruby/object:Overrides::ResourceOverrides
+      delete_minutes: 15
+    autogen_async: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime%s\", context[\"random_suffix\"])"
+        primary_resource_id: "runtime"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic_gpu"
+        primary_resource_id: "runtime_gpu"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime-gpu%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime-gpu"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic_container"
+        primary_resource_id: "runtime_container"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime-container%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime-container"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_full"
+        primary_resource_id: "runtime_full"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime-full%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime-full"
+    description: |
+      {{description}}
+
+      ~> **Note:** Due to limitations of the Notebooks Runtime API, many fields
+      in this resource do not properly detect drift. These fields will also not
+      appear in state once imported.
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants: templates/terraform/constants/notebooks_runtime.go
+    properties:
+      virtualMachine.virtualMachineConfig.labels: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+        diff_suppress_func: NotebooksRuntimeLabelDiffSuppress
+      virtualMachine.virtualMachineConfig.internalIpOnly: !ruby/object:Overrides::Terraform::PropertyOverride
+        send_empty_value: true
+      virtualMachine.virtualMachineConfig.metadata: !ruby/object:Overrides::Terraform::PropertyOverride
+        # API returns empty value.
+        default_from_api: true
+      virtualMachine.virtualMachineConfig.tags: !ruby/object:Overrides::Terraform::PropertyOverride
+        # API returns empty value.
+        default_from_api: true
+      virtualMachine.virtualMachineConfig.dataDisk.initializeParams.labels: !ruby/object:Overrides::Terraform::PropertyOverride
+        # API returns empty value.
+        default_from_api: true
+      virtualMachine.virtualMachineConfig.containerImages: !ruby/object:Overrides::Terraform::PropertyOverride
+        # API does not return this value.
+        diff_suppress_func: NotReturnedByAPIDiffSuppress
```

Comment:
@ScottSuarez , the containerImages are not returned by the API. I added this function to allows ignore but I am wondering if there's a more official way?

Reply:
You can just ignore_read on this field and that should work !

Reply:
Hmm, I am pretty sure I tried that and it didn't work but I'll try again. Thanks!

Reply:
Confirming that ignore_read does not work. I'll close it for now as the function seems to fix the problem and does not create others.